diff --git a/Makefile.am b/Makefile.am
index d0f54e5..e535329 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -89,8 +89,6 @@
 	weatherplugin \
 	weathercomponent \
 	weathercomponenthandler \
-	webadmin \
-	webbouqueteditor \
 	webcamviewer \
 	werbezapper \
 	zaphistorybrowser \
@@ -101,7 +99,6 @@
 	schiffe \
 	sudoku \
 	vierg \
-	bmediacenter \
 	advancedmovieselection
 
 if DREAMBOX
@@ -125,7 +122,5 @@
 SUBDIRS += \
 	mytube \
 	yttrailer \
-	webinterface \
 	menusort \
-	pluginhider \
 	pluginsort 

