From 92738c116d2b2579062983e949e1d30afbb25561 Mon Sep 17 00:00:00 2001
From: torwald12 <torwald12@team.nachtfalke.biz>
Date: Sun, 2 Sep 2018 18:42:36 +0200
Subject: [PATCH] revert-opkg-tools

---
 .../enigma2-plugins/enigma2-3rdparty-plugins.bb    |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/meta-oe/recipes-oe-alliance/enigma2-plugins/enigma2-3rdparty-plugins.bb b/meta-oe/recipes-oe-alliance/enigma2-plugins/enigma2-3rdparty-plugins.bb
index b1d1639..e76b0a8 100644
--- a/meta-oe/recipes-oe-alliance/enigma2-plugins/enigma2-3rdparty-plugins.bb
+++ b/meta-oe/recipes-oe-alliance/enigma2-plugins/enigma2-3rdparty-plugins.bb
@@ -35,7 +35,6 @@ THIRDPARTY_PLUGINS = " \
     enigma2-plugin-extensions-chefkoch_1.5rc5_all.ipk \
     enigma2-plugin-extensions-clevertanken_0.8rc1_all.ipk \
     enigma2-plugin-extensions-csfd_11-55-20160929_all.ipk \
-    enigma2-plugin-extensions-customsubservices_0.1.2_all.ipk \
     enigma2-plugin-extensions-cyrussettings_1.0.0_all.ipk \
     enigma2-plugin-extensions-digitalfernsehen_1.2rc1_all.ipk \
     enigma2-plugin-extensions-dvrproviderplayer_2.06_all.ipk \
@@ -49,8 +48,6 @@ THIRDPARTY_PLUGINS = " \
     enigma2-plugin-extensions-hetweer_4.2r9_all.ipk \
     enigma2-plugin-extensions-ipchecker_002_all.ipk \
     ${@bb.utils.contains("TARGET_ARCH", "sh4", "enigma2-plugin-extensions-isettinge2-3.3.9-oe2.0-all-sh4.ipk" , "", d)} \
-    ${@bb.utils.contains("TARGET_ARCH", "mipsel", "enigma2-plugin-extensions-isettinge2_3.4.1-oe2.0-all-mips_all.ipk" , "", d)} \
-    ${@bb.utils.contains("TARGET_ARCH", "arm", "enigma2-plugin-extensions-isettinge2-3.3.9-oe2.0-all-arm.ipk" , "", d)} \
     enigma2-plugin-extensions-kicker_3.6rc4_all.ipk \
     enigma2-plugin-extensions-kino_1.8rc1_all.ipk \
     enigma2-plugin-extensions-livefootball_7.1_all.ipk \
-- 
1.7.9.5

