From 03076384038ecd8297409f3f88bc5839d6643880 Mon Sep 17 00:00:00 2001
From: torwald12 <torwald12@team.nachtfalke.biz>
Date: Wed, 28 Feb 2018 18:53:12 +0100
Subject: [PATCH] revert-opkg-tools

---
 .../enigma2-plugins/enigma2-3rdparty-plugins.bb    |    2 --
 1 file changed, 2 deletions(-)

diff --git a/meta-oe/recipes-oe-alliance/enigma2-plugins/enigma2-3rdparty-plugins.bb b/meta-oe/recipes-oe-alliance/enigma2-plugins/enigma2-3rdparty-plugins.bb
index 609ad24..a4507d3 100644
--- a/meta-oe/recipes-oe-alliance/enigma2-plugins/enigma2-3rdparty-plugins.bb
+++ b/meta-oe/recipes-oe-alliance/enigma2-plugins/enigma2-3rdparty-plugins.bb
@@ -60,8 +60,6 @@ THIRDPARTY_PLUGINS = " \
     enigma2-plugin-extensions-mspfs_1.06_all.ipk \
     enigma2-plugin-extensions-netspeedtest_1.0rc1_all.ipk \
     ${@bb.utils.contains('MACHINE_BRAND', 'Vu+', '' , 'enigma2-plugin-extensions-openopera_1.0-r0_mips32el.ipk', d)} \
-    ${@bb.utils.contains("TARGET_ARCH", "mipsel", "enigma2-plugin-extensions-opkg-tools_1.4_mips32el.ipk" , "", d)} \
-    ${@bb.utils.contains("TARGET_ARCH", "arm", "enigma2-plugin-extensions-opkg-tools_1.4_cortexa15hf-neon-vfpv4.ipk" , "", d)} \
     enigma2-plugin-extensions-oscamstatusview_1.0rc1_all.ipk \
     enigma2-plugin-extensions-piconmanager_2.2-20150329-r0_all.ipk \
     enigma2-plugin-extensions-piconsupdater_0.3.7_mips32el.ipk \
-- 
1.7.9.5

