diff --git a/src/InfoBarSupport.py b/src/InfoBarSupport.py
index dc353d2..b04e0c6 100644
--- a/src/InfoBarSupport.py
+++ b/src/InfoBarSupport.py
@@ -101,6 +101,7 @@ class InfoBarSupport(	InfoBarBase, \
 											InfoBarPlugins, \
 											InfoBarNumberZap, \
 											InfoBarPiP, \
+		     									InfoBarHdmi, \
 											InfoBarEPG ):
 											#InfoBarMoviePlayerSummarySupport
 
@@ -115,7 +116,7 @@ class InfoBarSupport(	InfoBarBase, \
 				InfoBarSubtitleSupport, \
 				InfoBarTeletextPlugin, InfoBarServiceErrorPopupSupport, InfoBarExtensions, InfoBarNotifications, \
 				InfoBarPlugins, InfoBarNumberZap, \
-				InfoBarPiP, InfoBarEPG:
+				InfoBarPiP, InfoBarEPG, InfoBarHdmi:
 				#InfoBarCueSheetSupport
 				#InfoBarMoviePlayerSummarySupport
 			x.__init__(self)
@@ -335,4 +336,4 @@ class InfoBarSupport(	InfoBarBase, \
 			##InfoBarSeek.doSeek(self, end)
 			#InfoBarSeek._InfoBarSeek__evEOF(self)
 			#if self.service.type != sidDVB:
-			InfoBarSeek._InfoBarSeek__evEOF(self)
\ No newline at end of file
+			InfoBarSeek._InfoBarSeek__evEOF(self)
-- 
2.7.4
 
