From 71e937a2477033e4ec9ed2c20e6aa2486657383d Mon Sep 17 00:00:00 2001
From: carlo0815 <toto_golden@gmx.de>
Date: sat, 28 Dez 2019 20:30:49 +0200
Subject: [PATCH] add fileupload

---
 plugin/controllers/base.py | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/plugin/controllers/base.py b/plugin/controllers/base.py
index 2116a2e..cd1bb47 100644
--- a/plugin/controllers/base.py
+++ b/plugin/controllers/base.py
@@ -319,10 +319,12 @@ class BaseController(resource.Resource):
 			url = "%s://%s:%s" % (proto, request.getRequestHostname(), port)
 			if variant == "oscam":
 				extras.append({'key': url, 'description': _("OSCam Webinterface"), 'nw': '1'})
 			elif variant == "ncam":
 				extras.append({'key': url, 'description': _("NCam Webinterface"), 'nw': '1'})
+		url1 = "http://%s:8000" %request.getRequestHostname()
+		extras.append({'key': url1, 'description': _("FileLoad"), 'nw': '1'})		                
 
 		try:
 			from Plugins.Extensions.AutoTimer.AutoTimer import AutoTimer  # noqa: F401
 			extras.append({'key': 'ajax/at', 'description': _('AutoTimer')})
 		except ImportError:

