diff --git a/plugin/controllers/views/main.tmpl b/plugin/controllers/views/main.tmpl
index c73b024..04d83d6 100644
--- a/plugin/controllers/views/main.tmpl
+++ b/plugin/controllers/views/main.tmpl
@@ -200,7 +200,7 @@
 			<div id="content_container">

 			$content

 			</div>

-			<div id="footer"><h3>&nbsp;&nbsp;<a href="https://github.com/E2OpenPlugins">E2OpenPlugins</a> | <a href="http://www.opena.tv">openATV</a> | <a href="http://www.vuplus-community.net">Black Hole</a> | <a href="http://www.droidsat.org">OpenDroid</a> | <a href="http://www.egami-image.com">EGAMI</a> | <a href="http://www.hdfreaks.cc">OpenHDF</a> | <a href="http://image.openmips.com">openMips</a> | <a href="http://openpli.org">OpenPli</a> | <a href="http://forum.sifteam.eu">Sif</a> | <a href="http://openspa.info">OpenSpa</a> | <a href="http://www.world-of-satellite.com">OpenViX</a> | <a href="http://www.vuplus-support.org">VTi</a> | <a class="lmc" href="ajax/about">$tstrings['about']</a></h3></div>

+			<div id="footer"><h3>&nbsp;&nbsp;<a href="https://github.com/E2OpenPlugins">E2OpenPlugins</a> | <a href="http://www.nachtfalke.biz">openNFR</a> | <a href="http://www.opena.tv">openATV</a> | <a href="http://www.vuplus-community.net">Black Hole</a> | <a href="http://www.droidsat.org">OpenDroid</a> | <a href="http://www.egami-image.com">EGAMI</a> | <a href="http://www.hdfreaks.cc">OpenHDF</a> | <a href="http://image.openmips.com">openMips</a> | <a href="http://openpli.org">OpenPli</a> | <a href="http://forum.sifteam.eu">Sif</a> | <a href="http://openspa.info">OpenSpa</a> | <a href="http://www.world-of-satellite.com">OpenViX</a> | <a href="http://www.vuplus-support.org">VTi</a> | <a class="lmc" href="ajax/about">$tstrings['about']</a></h3></div>

 		</div>

 	</div>

 	<form name="portForm" action="web/stream.m3u" method="GET" target="_blank">

-- 
1.7.9.5
