diff --git a/plugin/controllers/base.py b/plugin/controllers/base.py
--- a/plugin/controllers/base.py
+++ b/plugin/controllers/base.py
@@ -357,6 +357,9 @@ def prepareMainTemplate(self, request):
			elif variant == "ncam":
				extras.append({'key': url, 'description': _("NCam Webinterface"), 'nw': '1'})

+		url1 = "http://%s:8000" %request.getRequestHostname()
+		extras.append({'key': url1, 'description': _("FileLoad"), 'nw': '1'})
+
		if HASAUTOTIMER:
			extras.append({'key': 'ajax/at', 'description': _('AutoTimers')})
