diff --git a/plugin/controllers/ajax.py b/plugin/controllers/ajax.py
index 4c661cd..057f749 100644
--- a/plugin/controllers/ajax.py
+++ b/plugin/controllers/ajax.py
@@ -326,7 +326,7 @@ class AjaxController(BaseController):
 		if transcoding:
 			try:
 				transcoder_port = int(config.plugins.transcodingsetup.port.value)
-				if getMachineBuild() in ('inihdp', 'hd2400', 'et10000', 'et13000', 'sf5008', 'ew7356', 'formuler1tc', 'tiviaraplus', '8100s'):
+				if getMachineBuild() in ('inihdp', 'hd2400', 'et10000', 'et13000', 'sf5008', 'ew7356', 'formuler1', 'tiviaraplus', '8100s'):
 					transcoder_port = int(config.OpenWebif.streamport.value)
 			except StandardError:
 				transcoder_port = 0
